includes/filerepo/file/LocalFile.php

```diff
@@ -1061,6 +1062,7 @@ function recordUpload2(
 				#throw new MWException( "Empty oi_archive_name. Database and storage out of sync?" );
 				Wikia::logBacktrace(__METHOD__ . "::oi_archive_name - [{$this->getName()}]"); // Wikia change (BAC-1068)
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)